tests/testthat/test_aggregate.R

```diff
@@ -1,3 +1,5 @@
+library(HDF5Array)
```

Comment:
instead of adding this here I would suggest to require the namespace for the package only in the unit tests were you use this package. See for example: [here](https://github.com/rformassspectrometry/MetaboAnnotation/blob/master/tests/testthat/test_CompDbSource.R#L43)

Also, I would then maybe manually install the `HDF5Array` in the github action (using `BiocManager::install("HDF5Array")`.